cmd/initialismer/main.go

```diff
@@ -0,0 +1,390 @@
+	"Cross-Origin-Embedder-Policy-Report-Only",
+	"Cross-Origin-Opener-Policy",
+	"Cross-Origin-Opener-Policy-Report-Only",
+	"Cross-Origin-Resource-Policy",
+	"DASL",
+	"Date",
+	"DAV",
+	"Default-Style",
+	"Delta-Base",
+	"Depth",
+	"Derived-From",
+	"Destination",
+	"Differential-ID",
+	"Digest",
+	"DPoP",
+	"DPoP-Nonce",
+	"Early-Data",
+	"EDIINT-Features",
+	"Expect",
+	"Expect-CT",
+	"X-Correlation-ID",
+	"X-UA-Compatible",
+	"X-XSS-Protection",
+	"Expires",
+	"Ext",
+	"Forwarded",
+	"From",
+	"GetProfile",
+	"Hobareg",
+	"Host",
+	"HTTP2-Settings",
+	"If",
+	"If-Match",
+	"If-Modified-Since",
+	"If-None-Match",
+	"If-Range",
+	"If-Schedule-Tag-Match",
+	"If-Unmodified-Since",
+	"IM",
+	"Include-Referred-Token-Binding-ID",
+	"Isolation",
+	"Keep-Alive",
+	"Label",
+	"Last-Event-ID",
+	"Last-Modified",
+	"Link",
+	"Location",
+	"Lock-Token",
+	"Man",
+	"Max-Forwards",
+	"Memento-Datetime",
+	"Meter",
+	"Method-Check",
+	"Method-Check-Expires",
+	"MIME-Version",
+	"Negotiate",
+	"NEL",
+	"OData-EntityId",
+	"OData-Isolation",
+	"OData-MaxVersion",
+	"OData-Version",
+	"Opt",
+	"Optional-WWW-Authenticate",
+	"Ordering-Type",
+	"Origin",
+	"Origin-Agent-Cluster",
+	"OSCORE",
+	"OSLC-Core-Version",
+	"Overwrite",
+	"P3P",
+	"PEP",
+	"PEP-Info",
+	"Permissions-Policy",
+	"PICS-Label",
+	"Ping-From",
+	"Ping-To",
+	"Position",
+	"Pragma",
+	"Prefer",
+	"Preference-Applied",
+	"Priority",
+	"ProfileObject",
+	"Protocol",
+	"Protocol-Info",
+	"Protocol-Query",
+	"Protocol-Request",
+	"Proxy-Authenticate",
+	"Proxy-Authentication-Info",
+	"Proxy-Authorization",
+	"Proxy-Features",
+	"Proxy-Instruction",
+	"Proxy-Status",
+	"Public",
+	"Public-Key-Pins",
+	"Public-Key-Pins-Report-Only",
+	"Range",
+	"Redirect-Ref",
+	"Referer",
+	"Referer-Root",
+	"Refresh",
+	"Repeatability-Client-ID",
+	"Repeatability-First-Sent",
+	"Repeatability-Request-ID",
+	"Repeatability-Result",
+	"Replay-Nonce",
+	"Reporting-Endpoints",
+	"Repr-Digest",
+	"Retry-After",
+	"Safe",
+	"Schedule-Reply",
+	"Schedule-Tag",
+	"Sec-GPC",
+	"Sec-Purpose",
+	"Sec-Token-Binding",
+	"Sec-WebSocket-Accept",
+	"Sec-WebSocket-Extensions",
+	"Sec-WebSocket-Key",
+	"Sec-WebSocket-Protocol",
+	"Sec-WebSocket-Version",
+	"Security-Scheme",
+	"Server",
+	"Server-Timing",
+	"Set-Cookie",
+	"Set-Cookie2",
+	"SetProfile",
+	"Signature",
+	"Signature-Input",
+	"SLUG",
+	"SoapAction",
+	"Status-URI",
+	"Strict-Transport-Security",
+	"Sunset",
+	"Surrogate-Capability",
+	"Surrogate-Control",
+	"TCN",
+	"TE",
+	"Timeout",
+	"Timing-Allow-Origin",
+	"Topic",
+	"Traceparent",
+	"Tracestate",
+	"Trailer",
+	"Transfer-Encoding",
+	"TTL",
+	"Upgrade",
+	"Urgency",
+	"URI",
+	"User-Agent",
+	"Variant-Vary",
+	"Vary",
+	"Via",
+	"Want-Content-Digest",
+	"Want-Digest",
+	"Want-Repr-Digest",
+	"Warning",
+	"X-Content-Type-Options",
+	"X-Frame-Options",
+	"ETag",
+	"DNT",
+	"X-Request-ID",
+	"X-XSS",
+	"X-DNS-Prefetch-Control",
+	"WWW-Authenticate",
+	"X-WebKit-CSP",
+	"X-Real-IP",
+}
```

Comment:
I would have defined only the ones where result would be different than well known. But I would have put all of them in the tests for testing possible regression.

Reply:
It's what you did in initialismer in fact, so I'm unsure why you have the well known list then 🤔

Reply:
I don't want to find manually non-canonical headers in the well-know list